packages/lesswrong/components/posts/PostsList2.tsx

```diff
@@ -209,6 +234,31 @@ const PostsList2 = ({
           }
         })}
       </div>
+      {commentsSection && isEAForum && currentUser?.profileTagIds?.length && <>
+        {commentsSection.loading ?
+          <Loading /> :
+          <>
+            {!!commentsSection.comments.length && <Typography
+              variant="body2" className={classes.commentSectionTitle}
+            >
+              <QuestionAnswerIcon className={classes.commentSectionIcon} />
+              {commentsSection.title}
+            </Typography>}
+            {commentsSection.comments.map((comment) => {
+              return <CommentsNode
+                treeOptions={{
+                  // F7U12
+                  tag: comment.tag ?? undefined,
+                  forceSingleLine: true
+                }}
+                comment={comment}
+                key={comment._id}
+                loadChildrenSeparately
+              />
+            })}
+          </>
+        }
+      </>}
```

Comment:
As someone has mentioned in slack I'm not sure about this appearing before load more, and if we don't want it there then this could live in HomeLatestPosts (I'm sure you've thought about this already)